common/utilities/number/stabilized-value.h

```diff
@@ -0,0 +1,103 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <string>
+#include <atomic>
+
+namespace utilities {
+namespace number {
+
+    // The stabilized value implement a history based stable value. 
+    // It supply a way to define and get values with a required stabilization.
+    // When the user adds a value, the value is inserted into the history
+    // When the user call "get" the last stable value is returned
+    // Inputs: Template <value_type>
+    //         required history size
+    //         required stabilization percentage
+    template<typename T> class stabilized_value
+{
+public:
+    stabilized_value( size_t history_size, float stabilize_percentage )
```

Comment:
I think maybe `stabilize_percentage` needs to have a default?
Also, if it's `percentage`, I would expect it to be between `0` and `100`. But you expect `0` to `1`, yes? In that case, it's a `fraction`

Reply:
I add an explanation on the API comments
required stabilization percentage as a fraction , range [0-1]
It is acceptable to threat percentage as 0-1 range, with the new comment the user should know what it means.